src/organization/organization.type.js

```diff
@@ -1,5 +0,0 @@
-import { createActionSet } from 'utils';
-
-export const GET_ORG = createActionSet('GET_ORG');
```

Comment:
I got rid of this `*.type.js` file in favor of `*.constants.js` and `*.actions.js` files